index.js

```diff
@@ -8,6 +8,18 @@ const schema = require('./src/schema/schema');
 const app = express();
 const PORT = process.env.PORT || 8000;
 
+const allowCrossDomain = function (req, res, next) {
```

Comment:
@ronanfitz because we are using the `cors` middleware (see below) I do not believe that we need to set all of these headers manually.